meetings/2022-11-07.md

```diff
@@ -0,0 +1,107 @@
+* eP: keep in mind that this is still a draft and if there are areas that need to be adjusted, we can ask if there are areas of the spec to address our needs.
+* ES: Can one really have OIDC Federation without having an OIDC server. With rolling out client identifiers to the general public, we saw developers to struggle even with that. Anything more complicated with that might be a step to far. We may need to provide a lot of tooling to make it possible to do for developers. We find many developers to host Client ID documents on solid storages.
+* AS: I think you hit the nail in the head. I was a little surprised how easy it was to mess up client identifiers. Using federation will be much more complicated, it's more complicated data structure and it also introduces signatures. There are 2 things that need to happen. With client identifiers part of the idea is that one can hand write those documents. At leeast that's how I did it. If we go OIDC Federation path we probably will want special tooling for it. There could be federation providers which could take care of the signing for the developer and we could provide a really good tooling for it.
+* eP: Agree that it will add complexity, but we also don't currently have a mechanism for trusted apps / app stores. There could be community / corporate trust anchors. There have been conversations to have a more centralized / review process for apps. So while there is complexity, it also provides a solution (partial?) to a problem that we haven't yet been able to solve.
+* WT: I agree that there could be some advantage. IMO even now developers need tooling to assist them.
+* AC: I agree with both of you. Yes it is more complicated but it also opens up to some features that we will need and don't have at this moment.
+* WT: Where in the spec it says that the leaf entity can't be their own trust anchor?
+* AC: Let me find it... https://openid.net/specs/openid-connect-federation-1_0.html#section-4.8
+* ...: It relates to federation fetch endpoint which has mutually exclusive requirements for trust anchor and leaf entity.
+* WT: Maybe it's just an oversight. 
+* eP: Perhaps we should create an issue for this. We can then get a sense of whether this might be addressed in the federation spec
+* AC: I think I have a good idea about the problem.
+
+ACTION: AC to create issue tracking disjont definition of leaf entity and trust 
+
+* ES: I understand that federation works with the same claims as dynamic registration. Is there a way to add more data? Let's say we have trus anchor at solidproject.org, can it say that all clients registered with it have a flag that it marks it as 'in preview' or untrusted. It would be helpful with any app marketplace to mark apps.
```

Comment:
```suggestion
* ES: I understand that federation works with the same claims as dynamic registration. Is there a way to add more data? Let's say we have trust anchor at solidproject.org; can it say that all clients registered with it have a flag that it marks as 'in preview' or untrusted? It would be helpful with any app marketplace to mark apps.
```